src/main/java/htsjdk/samtools/SAMUtils.java

```diff
@@ -87,27 +90,27 @@
     private static final byte COMPRESSED_K_HIGH = (byte) (COMPRESSED_K_LOW << 4);
     private static final byte COMPRESSED_D_HIGH = (byte) (COMPRESSED_D_LOW << 4);
     private static final byte COMPRESSED_B_HIGH = (byte) (COMPRESSED_B_LOW << 4);
-    
-    private static final byte [] COMPRESSED_LOOKUP_TABLE = 
+
+    private static final byte[] COMPRESSED_LOOKUP_TABLE =
             new byte[]{
```

Comment:
`new byte[]` is unnecessary here. If you remove it, I would pull the `{` back onto the previous line.